.travis.yml

```diff
@@ -7,6 +7,10 @@ services:
 
 matrix:
     include:
+     - python: 3.5
```

Comment:
I thought that pylint rules are the same for all versions. Do you know the differences?

Reply:
Not entirely sure, to be honest. Since the checks are fast I see no harm in running them explicitly rather than implicitly assuming there is no difference.

Reply:
I hope we'll never come across conflicting rules here ;)

Thanks for your contribution :+1: